tools/hidl/hidlcomp.h

```diff
@@ -958,6 +958,7 @@ class EspMethodInfo
 
         tags=NULL;
         next=NULL;
+        executionProfilingEnabled = false;
```

Comment:
For new member variables it's better to initialize them where they are declared in the body of the class.  Then the code improves over time even if we don't do a clean sweep and move all initialization there.